chef/.kitchen.yml

```diff
@@ -1,14 +1,15 @@
 ---
 driver:
   name: vagrant
-  provider: vmware_fusion
 
 provisioner:
   name: chef_solo
+  chef_omnibus_url: "https://gist.githubusercontent.com/netshade/28800461038c924b74a1/raw/a453b4758ccb9b3d20f95d5465f0754c0fa6ffc3/install.sh"
```

Comment:
The script is in there ( `omnibus.sh`) , the reason it's in a URL is because the underlying Kitchen CI stuff makes the assumption that the script is remotely hosted; otherwise I would have just had it import locally.